packages/newspringchurchapp/src/user-settings/Locations/__snapshots__/Locations.tests.js.snap

```diff
@@ -107,41 +107,28 @@ exports[`Display Native Map with Locations Render a native map view 1`] = `
                       }
                     }
                   >
-                    <View
-                      animate={null}
-                      customAnimate={null}
-                      onReady={false}
+                    <Image
```

Comment:
yea this file shouldn't have changed, something fishy is happening here

Reply:
@redreceipt @richarddubay My local snaps are messed up somehow. Y'all should run the snaps and update this branch.